### PR TITLE
Updated the ci-prod.yml for release

### PR DESCRIPTION
**Pull request by tahirjaved1**

### CODE REVIEW

File: .github/workflows/ci-prod.yml [hunk 1]
```diff
@@ -88,28 +88,47 @@ jobs:
...
-        
-    - name: Get Project version from Package.appxmanifest
-      id: project-version
-      shell: pwsh
-      run: |
-        [xml]$manifest = Get-Content -Path './Package.appxmanifest' # Make sure to update the path for this file
-        $version = $manifest.Package.Identity.Version
-        echo "::set-output name=current-version::$version"
+        name: MSIX Package x86
+        path: Packages\*.msix
```
Comment (gagansuie) [inline .github/workflows/ci-prod.yml:98]: this is running duplicate code. please revert back to using matrix.platform and delete the x86 step.

    - name: Upload MSIX package
      uses: actions/upload-artifact@v4
      with:
        name: MSIX Package ${{ matrix.platform }}
        path: Packages\*.msix

Comment (gagansuie) [inline .github/workflows/ci-prod.yml:98]: this is running duplicate code. please revert back to using matrix.platform and delete the x86 step.

    - name: Upload MSIX package
      uses: actions/upload-artifact@v4
      with:
        name: MSIX Package ${{ matrix.platform }}
        path: Packages\*.msix

Comment (gagansuie) [inline .github/workflows/ci-prod.yml]: this is running duplicate code. please revert back to using matrix.platform and delete the x86 step.

    - name: Upload MSIX package
      uses: actions/upload-artifact@v4
      with:
        name: MSIX Package ${{ matrix.platform }}
        path: Packages\*.msix

Comment (tahirjaved1) [reply .github/workflows/ci-prod.yml]: I have added the code and deleted the x86 step.

**Review by gagansuie — APPROVED** [review]

**Pull request merged**